packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -206,12 +228,13 @@ function NavigationMenuItemEdit( {
 
 export default compose( [
 	withSelect( ( select, ownProps ) => {
-		const { getClientIdsOfDescendants, hasSelectedInnerBlock } = select( 'core/block-editor' );
+		const { getClientIdsOfDescendants, hasSelectedInnerBlock, getSettings } = select( 'core/block-editor' );
 		const { clientId } = ownProps;
 
 		return {
 			isParentOfSelectedBlock: hasSelectedInnerBlock( clientId, true ),
 			hasDescendants: !! getClientIdsOfDescendants( [ clientId ] ).length,
+			fetchSearchSuggestions: getSettings().__experimentalFetchLinkSuggestions,
```

Comment:
This was removed in a previous commit, but seems to be back again.

Reply:
Fixed.